server/protocols/jmap/src/main/java/org/apache/james/jmap/model/mailbox/MailboxCreateRequest.java

```diff
@@ -76,12 +77,12 @@ public Builder parentId(MailboxCreationId parentId) {
 
         public Builder role(Role role) {
             Preconditions.checkNotNull(role);
-            throw new NotImplementedException();
+            throw new JmapFieldNotSupportedException(COMMAND, "role");
         }
 
         public Builder sortOrder(SortOrder sortOrder) {
             Preconditions.checkNotNull(sortOrder);
-            throw new NotImplementedException();
+            throw new JmapFieldNotSupportedException(COMMAND, "sortOrder");
```

Comment:
Did you forgot committing `JmapFieldNotSupportedException` ? I can not see it...

Reply:
Exactly